src/layouts/index.jsx

```diff
@@ -132,6 +151,7 @@ class Layout extends React.Component {
       totalQAsByPage: this.getTotalQAsByPage(),
       questionNumbersByPage: this.getQuestionNumbersByPage(),
       checkpoints: this.getCheckpoints(),
```

Comment:
checkpoints have sort of fallen by the wayside with the introduction of sections. I am open to removing the checkpoints code completely since it is now unused.